test-e2e/tests/watch_history/logged_in_test.ts

```diff
@@ -62,7 +40,8 @@ function runTestSuite(config: typeof testConfigs.svod, configNoWatchlist: typeof
   Scenario(`I can see my watch history on the Home screen when logged in - ${providerName}`, async ({ I }) => {
     I.dontSee(constants.continueWatchingShelfTitle);
 
-    await registerOrLogin(I);
+    await registerAndPlayVideo(I, videoTitle);
```

Comment:
You've also changed this test so it's not testing the restoration, only the storing within a session.